README.md

```diff
@@ -117,6 +117,18 @@ predictions = model.predict([
 print(predictions)
 ```
 
+### Serving
+
+Note: `Serve` is a framework agnostic serving engine ! [Learn more](flash_examples/serve/generic/boston_prediction/inference_server.py)
+
+```python
+from flash import FlashServeModel
```

Comment:
I suggest changing to `Serve`  -- it sounds more elegant than `FlashServeModel`. Then with the proper namespacing we can know what happens easily:

```python
import flash

flash.Serve()
```

Reply:
Changed for:

```
from flash.text import TranslationTask

model = TranslationTask.load_from_checkpoint("https://flash-weights.s3.amazonaws.com/translation_model_en_ro.pt")
model.serve()
```